src/pages/partySelectionContainer/partySelection/PartySelection.tsx

```diff
@@ -19,16 +19,23 @@ export default function PartySelection({ parties }: Props) {
   const { t } = useTranslation();
   const bodyTitle = t('partySelection.title');
   const bodyDescription = t('partySelection.subTitle');
-
-  const [selectedParty, setSelectedParty] = React.useState<Party | null>();
+  const [selectedParty, setSelectedParty] = React.useState<Party | null>(
+    parties.length === 1 ? parties[0] : null
+  );
   const [disableBtn, setBtnDisable] = React.useState(true);
   const history = useHistory();
   const dispatch = useAppDispatch();
 
   useEffect(() => {
     dispatch(partiesActions.setPartySelected(undefined));
     dispatch(partiesActions.setPartySelectedProducts(undefined));
-  }, []);
+
+    if (selectedParty) {
+      setBtnDisable(false);
+    } else {
+      setBtnDisable(selectedParty === null);
+    }
+  }, [selectedParty]);
```

Comment:
use a useEffect separated instead of change the condition of the pre-existent.

Moreover, ivoke setBtnDisable once using just the predicate on the defined selectedParty